avm/res/web/site/README.md

```diff
@@ -2118,7 +2118,7 @@ For all other app settings key-value pairs use this object.
 "appSettingsKeyValuePairs": {
     "value": {
       "AzureFunctionsJobHost__logging__logLevel__default": "Trace",
-      "EASYAUTH_SECRET": "https://adp-[[namePrefix]]-az-kv-x-001.vault.azure.net/secrets/Modules-Test-SP-Password",
+      "EASYAUTH_SECRET": "https://adp-#_namePrefix_#-az-kv-x-001.vault.azure.net/secrets/Modules-Test-SP-Password",
```

Comment:
@AlexanderSehr do we still have Parameter Usage content in READMEs?

Reply:
Hey @eriqua, only if it's part of the `## Notes`. @ahmadabdalla and I removed almost all and only kept a handful that we moved to the notes

Reply:
How to move forward? @eriqua @AlexanderSehr

Reply:
We should probably rediscuss this some time as this content is very likely to get stale. I'll approve the PR meanwhile as it won't harm

Reply:
Happy to. Kept it only where it seemed meaningful